pkg/registry/apis/iam/register.go

```diff
@@ -1,8 +1,21 @@
-package identity
+package iam
 
 import (
 	"context"
 
+	"github.com/grafana/grafana/pkg/apimachinery/identity"
+	identityv0 "github.com/grafana/grafana/pkg/apis/iam/v0alpha1"
```

Comment:
There are a bunch of these in import path. Should probably be iam or iamv0. Not a blocker for this one we can just update as we go